torchci/lib/bot/mergeBot.ts

```diff
@@ -44,6 +48,18 @@ function mergeBot(app: Probot): void {
         await reactOnComment(ctx, "confused");
         return;
       }
+      const revertWithReasonCmdPat = new RegExp(
+        "^\\s*@pytorch(merge|)bot\\s+revert\\s+this.*(\\s+\\w+){3,}"
+      );
+      if (commentBody.match(revertWithReasonCmdPat) === null) {
+        // revert reason of 3+ words not given
+        await addComment(
+          ctx,
+          "Revert unsuccessful: please retry the command explaining why the revert is necessary, " +
```

Comment:
might want something for 3+ words but guess it's fine. Or have them do like reason=blah so we can more easily parse out reasons down the line.

Reply:
Looking at the revert commands from the last 2 weeks, it looks like people have already been giving revert reasons in their own way so I don't think it'll be all that worth to switch it up and use reason=. We can just take the whole comment and it's usually good. https://fburl.com/scuba/opensource_github_webhook_events/p7fylx1x